### ISSUE TITLE
Create a new release for the guzzle6 branch

### ISSUE THREAD

**Issue by Jaesin** [guzzle/guzzle-services#118]
Is there anything left to be done before a tagged release of the guzzle6 branch can be created?

**Comment by Konafets** [comment]
The guzzle6 branch of [Guzzle Command](https://github.com/guzzle/command) need to be merged into master and a release has to be tagged first before Guzzle Service can follow. I am working at this topic.

**Comment by Jaesin** [comment]
OK, so waiting on : https://github.com/guzzle/command/issues/35

**Comment by Konafets** [comment]
Done!
